themes/socialblue/socialblue.info.yml

```diff
@@ -10,6 +10,7 @@ core: 8.x
 libraries:
   - socialblue/brand
   - socialblue/site-footer
+  - socialblue/nav-secondary--sky
```

Comment:
This means we're adding the library always, even when the sky flavour isn't used.